src/tests/Login.test.js

```diff
@@ -0,0 +1,44 @@
+import { fireEvent, render, waitFor } from "@testing-library/react";
+import axios from "axios";
+import { act } from "react-dom/test-utils";
+import { Login } from "../components/Login";
+
+const credentials = {
+  username: "jsmith@demo.io",
+  password: "Demo123!",
+};
+const testingToken = "SOME_TOKEN";
+const testingRole = "ROLE_USER";
+
+const mockedUsedNavigate = jest.fn();
+
+jest.mock("axios");
+
+jest.mock("react-router-dom", () => ({
+  ...jest.requireActual("react-router-dom"),
+  useNavigate: () => mockedUsedNavigate,
+}));
+
+test("Should access into the app after login", async () => {
+  const login = render(<Login />);
```

Comment:
I believe there's an `eslint` rule that recommends naming these constants _view_.

Reply:
view*

Yes!